src/zjs_error.c

```diff
@@ -161,7 +161,7 @@ jerry_value_t zjs_custom_error(const char *name, const char *message)
 jerry_value_t zjs_standard_error(zjs_error_type_t type, const char *message)
 {
     int count = sizeof(error_types) / sizeof(zjs_error_t);
-    if (type >= count || type < 0) {
+    if (type >= count) {
```

Comment:
Why? This seems safer, even if the enum type is unsigned. (I'm not sure... googling a little it appears gcc makes it unsigned by ISO standard says signed, so I'd stand by this being safer. We don't want to use a negative array index below.)